src/types/guild.ts

```diff
@@ -606,3 +607,29 @@ export interface CreateServerOptions {
   /** the id of the channel where guild notices such as welcome messages and boost events are posted */
   system_channel_id?: string;
 }
+
+// https://discord.com/developers/docs/resources/template#template-object
+export interface Template {
+  /** the template code (unique ID) */
+  code: string;
+  /** template name */
+  name: string;
+  /** the description for the template */
+  description: string | null;
+  /** number of times this template has been used */
+  usage_count: number;
```

Comment:
the template structure should try and conver these snake case to camelCase like the other structs